Add our own trash/delete button and hide gutenbergs.

StevenDufresne opened the pull request:
This PR, hide's Gutenbergs "Move to trash" button and shows our "Delete permanently" button.

## Screenshot
![](https://d.pr/i/uOKgD4.png)

StevenDufresne committed:
Add our own button and hide gutenbergs.

StevenDufresne committed:
Update the css to remove redundancy.

StevenDufresne committed:
The checkbox doesn't need to edit the post anymore.

coreymckrill reviewed (commented):
👍 Works for me.

* I'm thinking it might be worth adding an "Are you sure?" step in there. The button is big enough that an accidental click in the wrong place could be catastrophic.
* Weirdly, after I deleted the revision, it took me back to the Scheduled Updates list table, without reloading the page, so that the table still showed the revision I had just deleted. Is there a way to make it redirect to a URL after deleting so that the screen gets refreshed?

StevenDufresne commented:
> 👍 Works for me.
> 
> * I'm thinking it might be worth adding an "Are you sure?" step in there. The button is big enough that an accidental click in the wrong place could be catastrophic.
Yep. I can use the browser confirm dialog. It's the only confirmation panel that I have seen in Gutenberg. There is the fly out panel which is used for `publish` confirmation, but we can't use that panel and I don't think it's worth much effort.

> * Weirdly, after I deleted the revision, it took me back to the Scheduled Updates list table, without reloading the page, so that the table still showed the revision I had just deleted. Is there a way to make it redirect to a URL after deleting so that the screen gets refreshed?

Ah, it currently hits the `history.back()` browser api and I have never tested the list table path. I have only tested it from the original post. I'll look for a better move after delete.

StevenDufresne committed:
Add a confirmation window before delete.

StevenDufresne committed:
Update the copy and add domain for deleting update.

StevenDufresne committed:
Integrate controls into the document settings panel.

StevenDufresne committed:
Redirect to origin post after deleting.

StevenDufresne committed:
Revert "Redirect to origin post after deleting."

This reverts commit db9146bc810789e1f9c1147aa9b72ca69a3ba4b6.

StevenDufresne committed:
Remove unnecessary css style overrides.
